specs/vep-1243-vdk-meta-jobs/README.md

```diff
@@ -153,6 +188,107 @@ Consider at least the below topics but you do not need to cover those that are n
   * What secrets are needed by the components? How are these secrets secured and attained?
 -->
 
+### Workflow
+
+The structure of the Meta Job is a DAG. Here is an example of the DAG of jobs workflow:
+1. The Meta Job(DAG) is initialized, following the preset configuration.
+2. The DAG is being validated that there are no conflicts regarding the existence of the team/jobs in the
+Control Service it addresses as well as any other technical errors.
+3. If there are no errors during step 2., the DAG is built.
+4. The DAG of jobs is executed, taking into account all the order information.
+5. A summary of the DAG is logged.
+
+### Configuration details
+
+You are allowed to specify whether the Meta Job itself should fail if any of its orchestrated jobs encounter errors.
+This is achieved by adding the Data Job attribute **fail_meta_job_on_error**. Set to true, as by default, would mean
+that in the event of a failing orchestrated job, the Meta Job would also fail - with USER error.
+This can help to prevent cascading failures and ensure that issues are isolated and addressed as soon as possible.
+
+There are more settings that are configurable by the user, that can be explored in the
+[config](/projects/vdk-plugins/vdk-meta-jobs/src/vdk/plugin/meta_jobs/meta_configuration.py).
+
+### Capacity Estimation and Constraints
+
+The load to the Control Service APIs from the Meta Jobs plugin mostly depends on two factors:
+
+* Number of concurrent Data Job Executions (there is a reasonable default limit, but the value is
+  [configurable](https://github.com/vmware/versatile-data-kit/blob/main/projects/vdk-plugins/vdk-meta-jobs/src/vdk/plugin/meta_jobs/meta_configuration.py#L87);
+* Data Job Executions duration;
+* Configuration value of [META_JOBS_TIME_BETWEEN_STATUS_CHECK_SECONDS](https://github.com/vmware/versatile-data-kit/blob/main/projects/vdk-plugins/vdk-meta-jobs/src/vdk/plugin/meta_jobs/meta_configuration.py#L76).
+
+A workflow that consists of one Data Job is going to perform 2 requests per minute to the
+Control Service APIs.
+
+For example, the following workflow consists of 6 Data Jobs:
```

Comment:
Move the actual calcualtion into Appendix section (make new section at the end) and only leave the conclusion here . 

It's make it easier to follow the vep. The conclusion is the important part for most readers.